Keas.Mvc/Controllers/Api/EquipmentController.cs

```diff
@@ -112,6 +112,12 @@ public async Task<IActionResult> Create([FromBody]Equipment equipment)
                    var space = await _context.Spaces.SingleAsync(x => x.RoomKey == equipment.Space.RoomKey);
                     equipment.Space = space;
                 }
+
+                if (equipment.Type.Equals("Card", StringComparison.OrdinalIgnoreCase))
```

Comment:
Do we need to do this?  What happens in the other cases, some null error?

Reply:
@srkirkland We don't need it, but I figured we should clear out any values in case they are displayed somewhere else.

Reply:
hmm.. I don't like the double logic of it here, which I think will get out of hand pretty quickly when more types are added.  Either we should leave it alone (my thought) or in react, when type is changed we clear out fields which shouldn't apply (similar to your shouldShow method)